_source/_posts/2019-09-19-nodejs-typescript.md

```diff
@@ -0,0 +1,637 @@
+---
+layout: blog_post
+title: "Build a NodeJS App with TypeScript"
```

Comment:
Score is a 57 on our title analyzer.

Reply:
I've rarely seen NodeJS. Node.js or just Node seems more popular. Should this be NestJS instead? Otherwise, it seems like similar keywords to https://developer.okta.com/blog/2019/05/07/nodejs-typescript-api.